### PR TITLE
Fix for MultiUploadMetaInput widget under django 2.1

### PR DESCRIPTION
**Pull request by mtskelton**

### CODE REVIEW

File: multiupload/fields.py [hunk 1]
```diff
@@ -20,11 +20,11 @@ def __init__(self, *args, **kwargs):
         self.multiple = kwargs.pop('multiple', True)
         super(MultiUploadMetaInput, self).__init__(*args, **kwargs)
 
-    def render(self, name, value, attrs=None):
+    def render(self, name, value, attrs=None, renderer=None):
         if self.multiple:
             attrs['multiple'] = 'multiple'
 
-        return super(MultiUploadMetaInput, self).render(name, value, attrs)
+        return super(MultiUploadMetaInput, self).render(name, value, attrs=attrs, renderer=renderer)
```
Comment (goromachine) [inline multiupload/fields.py]: This line must be:
`        return super(MultiUploadMetaInput, self).render(name, value, attrs=None, renderer=None)
`
Please solve this and let's try to merge to have 2.1 compatibility.

Comment (mtskelton) [reply multiupload/fields.py]: Made and pushed the requested change.

Comment (nollidnosnhoj) [reply multiupload/fields.py]: doesn't leaving `attrs` to `None` removes multiple file uploading?

**Comment by nollidnosnhoj** [comment]
I tested out this pull request, and `attrs=None` removes the multiple attribute in the file input.

**Comment by Chive** [comment]
Closed in favour of #32

**Pull request closed without merging**